### ISSUE TITLE
Only instances of `keras.Layer` can be added to a Sequential model.

### ISSUE THREAD

**Issue by YonaniCodes** [mrdbourke/zero-to-mastery-ml#125]
` # Setup the model layers

  model = tf.keras.Sequential([
    hub.KerasLayer(MODEL_URL), # Layer 1 (input layer)
    tf.keras.layers.Dense(units=OUTPUT_SHAPE,
                          activation="softmax") # Layer 2 (output layer)
  ])`  

the input layer is not working  for me

**Comment by 0quaaD** [comment]
Hi! Have you ever tried to use **Functional API** instead of **Sequential** at 
`tf.keras.Sequential()` ?  
Because in your case, `hub.KerasLayer(MODEL_URL)` expects some string input. But in functional API:  
```python
inputs = tf.Keras.Input(shape=(), dtype = tf.string)
x =hub.KerasLayer(MODEL_URL)(inputs)
```
You can tell TensorFlow exactly what type of input is coming and how to pass it.

**Comment by YonaniCodes** [comment]
Thanks!!! I down graded the version and now it's working.

**Comment by Ann-Z-West** [comment]
Wrap it up with `lambda` function: `tf.keras.layers.Lambda(hub.KerasLayer(model_url))`

Complete function:
```
# Create a function which builds a Keras model
def create_model(input_shape=INPUT_SHAPE, output_shape=OUTPUT_SHAPE, model_url=MODEL_URL):
  print("Building model with:", MODEL_URL)

  # Setup the model layers
  model = tf.keras.Sequential([
    tf.keras.layers.Lambda(hub.KerasLayer(model_url)), # Layer 1 (input layer)
    tf.keras.layers.Dense(units=OUTPUT_SHAPE, 
                          activation="softmax") # Layer 2 (output layer)
  ])

  # Compile the model
  model.compile(
      loss=tf.keras.losses.CategoricalCrossentropy(), # Our model wants to reduce this (how wrong its guesses are)
      optimizer=tf.keras.optimizers.Adam(), # A friend telling our model how to improve its guesses
      metrics=["accuracy"] # We'd like this to go up
  )

  # Build the model
  model.build(INPUT_SHAPE) # Let the model know what kind of inputs it'll be getting
  
  return model
```

**Comment by YonaniCodes** [comment]
Thanks bro
